packages/flutter/lib/src/widgets/editable.dart

```diff
@@ -430,3 +463,153 @@ class _EditableLineWidget extends LeafRenderObjectWidget {
     return new TextSpan(style: style, text: text);
   }
 }
+
+enum _TextSelectionHandleSide { left, right }
```

Comment:
Probably should be `start` and `end`.  Otherwise you're going to end up with an LTR bias.